lib/controllers/ServiceFabrikApiController.js

```diff
@@ -490,12 +490,15 @@ class ServiceFabrikApiController extends BaseController {
           .then(instance => _.set(req, 'entity.name', instance.entity.name));
       })
       .then(() => req.manager.findDeploymentNameByInstanceId(req.params.instance_id))
-      .then(deploymentName => ({
-        instance_id: req.params.instance_id,
-        instance_name: req.entity.name,
-        deployment_name: deploymentName,
-        timeZone: req.body.timeZone
```

Comment:
why is timeZone removed ? We dont use it ?

Reply:
timezone is part of res.body which along with any other attribs coming in request is assisgned as part of job data. Look at inserted code segment.

Reply:
Ok , got it .